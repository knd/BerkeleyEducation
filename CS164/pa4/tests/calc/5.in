1 / 2 / 3 / 4 / m / s / 1 / 2 / m / s
