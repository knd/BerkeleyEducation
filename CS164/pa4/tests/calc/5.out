[(0, 1, 'Expr -> Fraction'),
 (0, 1, 'Fraction -> Number'),
 (0, 1, 'Number -> [0-9]+'),
 (0, 3, 'Expr -> Expr \\/ Expr'),
 (0, 5, 'Expr -> Expr \\/ Expr'),
 (0, 7, 'Expr -> Expr \\/ Expr'),
 (0, 9, 'Expr -> Expr \\/ Expr'),
 (0, 11, 'Expr -> Expr \\/ Expr'),
 (0, 13, 'Expr -> Expr \\/ Expr'),
 (0, 15, 'Expr -> Expr \\/ Expr'),
 (0, 17, 'Expr -> Expr \\/ Expr'),
 (0, 19, 'Expr -> Expr \\/ Expr'),
 (2, 3, 'Expr -> Fraction'),
 (2, 3, 'Fraction -> Number'),
 (2, 3, 'Number -> [0-9]+'),
 (2, 5, 'Expr -> Expr \\/ Expr'),
 (2, 7, 'Expr -> Expr \\/ Expr'),
 (2, 9, 'Expr -> Expr \\/ Expr'),
 (2, 11, 'Expr -> Expr \\/ Expr'),
 (2, 13, 'Expr -> Expr \\/ Expr'),
 (2, 15, 'Expr -> Expr \\/ Expr'),
 (2, 17, 'Expr -> Expr \\/ Expr'),
 (2, 19, 'Expr -> Expr \\/ Expr'),
 (4, 5, 'Expr -> Fraction'),
 (4, 5, 'Fraction -> Number'),
 (4, 5, 'Number -> [0-9]+'),
 (4, 7, 'Expr -> Expr \\/ Expr'),
 (4, 9, 'Expr -> Expr \\/ Expr'),
 (4, 11, 'Expr -> Expr \\/ Expr'),
 (4, 13, 'Expr -> Expr \\/ Expr'),
 (4, 15, 'Expr -> Expr \\/ Expr'),
 (4, 17, 'Expr -> Expr \\/ Expr'),
 (4, 19, 'Expr -> Expr \\/ Expr'),
 (6, 7, 'Expr -> Fraction'),
 (6, 7, 'Fraction -> Number'),
 (6, 7, 'Number -> [0-9]+'),
 (6, 9, 'Expr -> Expr \\/ Expr'),
 (6, 11, 'Expr -> Expr \\/ Expr'),
 (6, 13, 'Expr -> Expr \\/ Expr'),
 (6, 15, 'Expr -> Expr \\/ Expr'),
 (6, 17, 'Expr -> Expr \\/ Expr'),
 (6, 19, 'Expr -> Expr \\/ Expr'),
 (8, 9, 'Expr -> Unit'),
 (8, 9, 'Unit -> m'),
 (8, 11, 'Expr -> Expr \\/ Expr'),
 (8, 13, 'Expr -> Expr \\/ Expr'),
 (8, 15, 'Expr -> Expr \\/ Expr'),
 (8, 17, 'Expr -> Expr \\/ Expr'),
 (8, 19, 'Expr -> Expr \\/ Expr'),
 (10, 11, 'Expr -> Unit'),
 (10, 11, 'Unit -> s'),
 (10, 13, 'Expr -> Expr \\/ Expr'),
 (10, 15, 'Expr -> Expr \\/ Expr'),
 (10, 17, 'Expr -> Expr \\/ Expr'),
 (10, 19, 'Expr -> Expr \\/ Expr'),
 (12, 13, 'Expr -> Fraction'),
 (12, 13, 'Fraction -> Number'),
 (12, 13, 'Number -> [0-9]+'),
 (12, 15, 'Expr -> Expr \\/ Expr'),
 (12, 17, 'Expr -> Expr \\/ Expr'),
 (12, 19, 'Expr -> Expr \\/ Expr'),
 (14, 15, 'Expr -> Fraction'),
 (14, 15, 'Fraction -> Number'),
 (14, 15, 'Number -> [0-9]+'),
 (14, 17, 'Expr -> Expr \\/ Expr'),
 (14, 19, 'Expr -> Expr \\/ Expr'),
 (16, 17, 'Expr -> Unit'),
 (16, 17, 'Unit -> m'),
 (16, 19, 'Expr -> Expr \\/ Expr'),
 (18, 19, 'Expr -> Unit'),
 (18, 19, 'Unit -> s')]
Yes
